tensorflow_addons/optimizers/tests/rectified_adam_test.py

```diff
@@ -209,3 +210,26 @@ def test_scheduler_serialization():
         "class_name": "InverseTimeDecay",
         "config": wd_scheduler.get_config(),
     }
+
+
+def test_checkpoint_serialization():
```

Comment:
We get pytest fixture to access tmpdir. Try

https://github.com/tensorflow/addons/blob/9dd2389124a52e6cc298eba7d16c6732ec484d30/tensorflow_addons/layers/tests/spatial_pyramid_pooling_test.py#L67-L80